### DISCUSSION TITLE
VSC "User not authorized"

### DISCUSSION THREAD

**Discussion by k1lgor** [community/community#17300] [Copilot Conversations] [answered]
So, I just got the invite, had CoPilot already installed, and it gave me this error 
```
[INFO] [auth] [2022-05-26T09:01:30.918Z] Invalid copilot token: missing token: 404 
[ERROR] [default] [2022-05-26T09:01:30.920Z] GitHub Copilot could not connect to server. Extension activation failed: "User not authorized"
```
I have tried the same steps from [this](https://github.com/github/feedback/discussions/13448) post but was unsuccessful.

**Comment by davecheney** [comment, marked as answer]
@k1lgor i've verified your invite is active. Which version of the copilot extension are you using? Have you tried upgrading?

**Reply by k1lgor** [reply]
The installed version is v1.23.5969
[from here](https://marketplace.visualstudio.com/items?itemName=GitHub.copilot) as per documentation

**Reply by davecheney** [reply]
<img width="475" alt="Screen Shot 2021-11-30 at 8 03 57 am" src="https://user-images.githubusercontent.com/7171/143942796-0c2b6d34-e0f4-46fe-b349-f26087b2c5ed.png">
Please try signing out, copilot will prompt you to sign back in.

**Reply by k1lgor** [reply]
I have tried it 2-3 times, and the same result, same steps mentioned in the [post](https://github.com/github/feedback/discussions/13448)

**Reply by davecheney** [reply]
Can you please confirm you are using VSCode, not vscodium or oss code.

**Reply by k1lgor** [reply]
VSCodium

**Reply by davecheney** [reply]
I'm sorry, VScodium is not supported. You need to use VSCode.

**Reply by k1lgor** [reply]
Is it supported for Code - OSS?

**Reply by davecheney** [reply]
Only VScode is supported at the moment.
